pymeasure/adapters/usbtmc.py

```diff
@@ -0,0 +1,170 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2016 PyMeasure Developers
+#
+# Python USBTMC driver, lifted from python-usbtmc.
```

Comment:
What does "lifted" mean? If copied directly, maybe it's better to import-and-extend instead of copying source code (as you won't get upstream fixes/features that way). If not, maybe worth it to clarify that, and the actual license that was under, in the source.